makefiles/testing.mak

```diff
@@ -8,8 +8,8 @@ MAN_TEST_TMPDIR = ManTest
 check: tmain units tlib man-test check-genfile tutil
 
 # We may use CLEANFILES, DISTCLEANFILES, or etc.
-# clean-tlib and clean-gcov are not included
```

Comment:
@hirooih Can you remember why you didn't add `clean-tlib` to the dependency list of `clean-local`?

Reply:
> Can you remember why you didn't add clean-tlib to the dependency list of clean-local?

No, I cannot.

It is part of #3119 in which I made many changes for build system.

```diff
- clean-local: clean-units clean-tmain
+ # We may use CLEANFILES, DISTCLEANFILES, or etc.
+ # clean-tlib and clean-gcov are not included
+ clean-local: clean-units clean-tmain clean-man-test
```

I only added `clean-man-test`, but not `clean-tlib` or `clean-gcov`.

Here is my guess.

- After making changes for this PR I run `make distclean` and found I had to add `clean-man-test`.
- I found that `clean-tlib` and `clean-gcov` were not included, either.
- I was not sure why they were not included because I never need to use them.  (I did not run gcov and I don't know when clean-tlib is needed even now.)
- I left the comment to notify that to you.
- I forgot to mention about it in my PR message.

I hope this helps.

***

It seems that you have finally implemented the reference tags for C parser. Great!

Reply:
Thank you. I will add clean-tlib or clean-gcov to the list as a trial.